www/%team/distributing/index.html.spt

```diff
@@ -1,10 +1,19 @@
 # encoding: utf8
+
+from aspen import Response
 from gratipay.utils import get_team
 
 [-----------------------------------------------------------------------------]
 team = get_team(state)
 if team.available == 0:
     website.redirect('..', base_url='')
+
+if user.participant == None:
+    raise Response(401)
+
+if not (user.ADMIN or user.participant.member_of(team) or user.participant.username == team.owner):
+    raise Response(403)
+
```

Comment:
Oh I missed your comment… I was thinking that since there were no link to this page for the ordinary user, it was not supposed to be reachable.